### PR TITLE
extract oracle plugin

### PR DESCRIPTION
**Pull request by SwampDragons**
moved to https://github.com/hashicorp/packer-plugin-oracle

### CODE REVIEW

**Review by sylviamoss — APPROVED** [review]
LGTM! 👍🏼

File: website/data/docs-remote-plugins.json [hunk 1]
```diff
@@ -93,6 +93,13 @@
     "pluginTier": "community",
     "version": "latest"
   },
+  {
+    "title": "Oracle",
+    "path": "oracle",
+    "repo": "hashicorp/packer-plugin-oracle",
+    "version": "latest",
+    "pluginTier": "community"
```
Comment (sylviamoss) [inline website/data/docs-remote-plugins.json:101]: Community or partner? I'm not sure

**Comment by prydie** [comment]
@SwampDragons Are you adding the existing code owners?

**Comment by sylviamoss** [comment]
@prydie Yes. I'll be doing that today.

**Comment by prydie** [comment]
@sylviamoss We've got an [open PR](https://github.com/hashicorp/packer-plugin-oracle/pull/16) but I haven't been added to the repo so I'm unable to review / merge it. Are you able to add the code owners to the new repo ASAP please?

**Comment by SwampDragons** [comment]
Hi! It looks like I don't have access to add non-HashiCorp users to the repo, but I can merge for you if you've reviewed. 

If you're interested, we can also transfer ownership of this plugin to Oracle so you can maintain it more easily. Docs would still live on Packer.io, but Oracle would have full control of the plugin.

**Pull request merged**